tools/monitoring/cloudmail/setup/setup.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/knative/test-infra/tools/monitoring/cloudmail"
+)
+
+func main() {
+	actionCreateDomain := flag.Bool("setup-domain", false, "Create the cloud mail domain")
+	actionSetupSender := flag.Bool("setup-sender", false, "Setup the address set, sender, receipt rule with default settings")
+	actionSetupAll := flag.Bool("setup-all", false, "Setup up the domain, address set, sender, receipt rules with default settings")
+	actionSendTestMail := flag.Bool("send-test-mail", false, "Send a test message")
+
+	toAddr := flag.String("to-address", "", "The test email recipient address")
+
+	flag.Parse()
+
+	ctx := context.Background()
+	client, err := cloudmail.NewMailClient(ctx)
+	if err != nil {
```

Comment:
You don't need the `if` if `failIfError` does nothing when `err` is `nil`.

Reply:
Removed the if statement